elastic-mapping-updater/src/main/java/com/github/cafdataprocessing/elastic/tools/ElasticMappingUpdater.java

```diff
@@ -308,30 +360,6 @@ private static String getParamName(final String key)
                 || ((Map<?, ?>) (e.getValue().rightValue())).containsKey(MAPPING_TYPE_KEY)))
                 .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue()));
 
-            // Check if any fields have parameters being removed that may not be reflected in the index mapping update
-            final Map<String, ValueDifference<Object>> entriesBeingRemoved = entriesDiffering.entrySet().stream()
-                .filter(e -> !typeDifferences.containsKey(e.getKey()))
-                .filter(e -> ((Map<?, ?>) (e.getValue().rightValue())).size() > ((Map<?, ?>) (e.getValue().leftValue())).size())
-                .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue()));
-            if (!entriesBeingRemoved.isEmpty()) {
-                entriesBeingRemoved.forEach(
-                    (key, value) -> {
-                        LOGGER.warn(
-                                "Mapping changes with some parameters being removed may not be applied for : {} - current: {} target: {}",
-                                key, value.rightValue(), value.leftValue());
-                        //Find the properties removed in the update
-                        final Sets.SetView<?> elementsRemoved = Sets.difference(
-                                ((Map<?, ?>) (value.rightValue())).keySet(),
-                                ((Map<?, ?>) (value.leftValue())).keySet()
-                        );
-                        //If one of the removed properties is unmodifiable, disable this update.
-                        if (!MODIFIABLE_PROPERTIES.containsAll(elementsRemoved)) {
-                            allowedFieldDifferences.remove(key);
-                        }
-                    });
-                unsupportedObjectChanges = true;
-            }
-
```

Comment:
Why is this section being removed?

Reply:
Better handled when other unsupported changes are being detected in the isMappingChangeSafe function:
type change
adding un-modifiable params
removing un-modifiable params